src/posh-git.psm1

```diff
@@ -87,7 +87,8 @@ if ($ForcePoshGitPrompt -or !$currentPromptDef -or ($currentPromptDef -eq $defau
         Write-VcsStatus
 
         # If stopped in the debugger, the prompt needs to indicate that in some fashion
-        $debugMode = (Test-Path Variable:/PSDebugContext) -or [runspace]::DefaultRunspace.Debugger.InBreakpoint
+        $hasInBreakpoint = [runspace]::DefaultRunspace.Debugger | Get-Member -Name InBreakPoint -MemberType property
```

Comment:
Sorry, one more minor nit.  The name the `p` in `Breakpoint` should be lower case everywhere.  There is one instance on this line and one on the next line.  Again, sorry I didn't catch this the first time.